client/rpc-servers/src/lib.rs

```diff
@@ -91,24 +91,23 @@ mod inner {
 						return;
 					}
 				};
-
+				// TODO: (dp) DRY this up; it's the same as the WS code
 				let handle = server.stop_handle();
-
-				server.register_module(module).expect("infallible already checked; qed");
 				let mut methods_api = RpcModule::new(());
-				let mut methods = server.method_names();
-				methods.sort();
+				let mut available_methods = module.method_names().collect::<Vec<&'static str>>();
```

Comment:
`collect` only needs to know the container type, item will get inferred.

```suggestion
				let mut available_methods = module.method_names().collect::<Vec<_>>();
```